src/cli/main.ts

```diff
@@ -0,0 +1,44 @@
+import yargs from 'yargs/yargs';
+import {Worker} from 'worker_threads';
+import {CommandProcessor} from './command-processor';
+
+const APP_INSTANCE_PATH = './build/src/cli/app-instance.js';
+
+/**
+ * Process initial command-line arguments.
+ */
+const argv = yargs()
+  .usage('Usage $0 --app_path PATH')
+  .options('app_path', {
+    describe: 'The path of the Local Fulfillment App entry point.',
+    type: 'string',
+    demandOption: true,
+  })
+  .parse(process.argv.slice(2));
+
+/**
+ * Entry point for running the command line interface.
+ */
+async function main() {
+  const worker = new Worker(APP_INSTANCE_PATH, {
+    workerData: argv.app_path,
+  });
+
+  worker.on('error', error => {
+    console.log(
+      'An error occured while trying to initialize the command line interface:\n' +
+        error.toString()
+    );
+  });
+
+  worker.on('message', async message => {
+    if (message === 'ready') {
```

Comment:
Is 'ready' a known constant based on an enum?

Reply:
It's not built in, but it should at least be a constant rather than a literal.  I added a declaration to `command-processor.ts`